INN/umbrella-ipbs
# Styling for homepage video widget

## Changes

This pull request makes the following changes:

- Adds basic styling for a video area on the homepage

Desktop:
![Screen Shot 2020-02-03 at 10 10 47 AM](https://user-images.githubusercontent.com/18353636/73664489-80880980-466d-11ea-8888-0d23c06ebd67.png)

Mobile:
![Screen Shot 2020-02-03 at 10 10 58 AM](https://user-images.githubusercontent.com/18353636/73664488-80880980-466d-11ea-8e21-5ca46c07eefa.png)

## Why



For #60

## Testing/Questions

Features that this PR affects:

- Homepage video area


Questions that need to be answered before merging:

- [ ] Do we want to spend the extra time styling the video play button to match the mockup? I know we're trying for the quickest solution here, so this PR does not include that.

Steps to test this PR:

1. Add a custom hmtl widget (or text) to the homepage bottom widget area and paste this:
```
<div class="video-widget">
	<div class="video-widget-content-left span5">
		<h2>I Love IPBS Video</h2>
		<p>Qui officia deserunt mollit anim id est laborum. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusan.</p>
	</div>
	<div class="video-widget-content-right span7">
		<iframe src="https://www.youtube.com/embed/s4hBAQHSU9o" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
	</div>
</div>
```
2. View on desktop and mobile and make sure it looks ok
